src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -524,3 +539,31 @@ def update_os_patch_configuration_sub_setting(self, patch_configuration_sub_sett
     def do_processes_require_restart(self):
         """Defaulting this for Apt"""
         return False
+
+    def is_reboot_pending(self):
```

Comment:
Just a heads up: because of an unrelated bug in assessment, I've delegated this function down to each package manager from the base class. That does not conflict with the direction you are taking but you'll need to reconcile and combine the function bodies. It shouldn't affect anything you were going to do - just the merge.

Reply:
Got it. Thanks for the update.